mockgcp/mocksecretmanager/secrets.go

```diff
@@ -84,6 +84,13 @@ func (s *SecretsV1) populateDefaultsForSecret(ctx context.Context, obj *pb.Secre
 			return fmt.Errorf("Aliases cannot be assigned to versions that don't exist")
 		}
 	}
+	// TTL and ExpireTime are OneOf, but the GCP service always converts TTL to expireTime before storing the object.
```

Comment:
This is a reasonable behaviour.  Maybe we should deprecate spec.ttl (and replace with `spec.initialTTL` - although if we are going to do that, we can just change the comment on the field)